### PR TITLE
Bufix/3061 crlf fix

### PR DESCRIPTION
**Pull request by davidnich**

### CODE REVIEW

File: lib/QC_File.qpp [hunk 1]
```diff
@@ -64,10 +64,18 @@ QoreFile* QoreFileHelper::operator->() const {
...
-   //printd(5, "FILE_system_constructor() self=%p, descriptor=%d\n", self, fd);
-   File* f = new File(QCS_DEFAULT);
-   f->makeSpecial(fd);
-   self->setPrivate(CID_FILE, f);
+    //printd(5, "FILE_system_constructor() self=%p, descriptor=%d\n", self, fd);
+    File* f = new File(QCS_DEFAULT);
+    f->makeSpecial(fd);
+    self->setPrivate(CID_FILE, f);
+#ifdef _Q_WINDOWS
+    // we have to set inary mode on stdin/stdout/stderr so that \r\n is not automatically
```
Comment (pvanek) [inline lib/QC_File.qpp]: s/inary/binary

**Review by pvanek — CHANGES_REQUESTED** [review]

**Comment by pvanek** [comment]
BTW is there some HOWTO for Windows compilation, please?

File: lib/QC_File.qpp (hunk 1 above)
Comment (davidnich) [reply lib/QC_File.qpp]: fixed

**Comment by davidnich** [comment]
@pvanek there is not, we can make a wiki page for it

I'm using `qrb` with the following script that I call `wqrb`:
```
#!/bin/bash

export CMAKE=x86_64-w64-mingw32.static-cmake

export MPFR_DIR=/opt/mxe64
export OPENSSL_DIR=$HOME/src/mxe/usr/x86_64-w64-mingw32.static
export LIBSSH2_DIR=/opt/mxe64
#export LIBSSH2_WINCNG=1
unset LIBXML2_DIR

export PGSQL_DIR=/opt/mxe64/pgsql
export MYSQL_DIR=/opt/mxe64/mariadb
unset MYSQL_INCLUDES
unset MYSQL_LIBS
export ORACLE_INSTANT_CLIENT=/opt/mxe64/oracle/instantclient_12_1

export CXX=
export CC=

export DEV=~/src/windows

export QORE_GIT_ROOT=$DEV/qore/git
export QORUS_GIT_ROOT=$DEV/Qorus/git

export OMQ_DIR=~/src/Qorus/wintest

export QORE_INCLUDE_DIR=$OMQ_DIR/qlib
export QORE_MODULE_DIR=$OMQ_DIR/qlib
export QORE_INSTALL_PREFIX=/opt/mxe64
export QORE_LIBRARY=/opt/mxe64/bin/libqore-5.dll

export QPP=qpp

export PATH=$OMQ_DIR/bin:$PATH

if [ -z "`echo $* | grep -e -l`" ]; then
    qrb_args="-kjni,odbc,process"
fi

qrb $qrb_args -W $*
```

**Review by pvanek — APPROVED** [review]

**Pull request merged**